### ISSUE TITLE
Under search "Angebote" field of search Area "von Grösse" is not defined.

### ISSUE THREAD

**Issue by Dummylearners** [AkrosAG/Akros-Marketplace#287]
-  when size of the Zimmer or any Apartment is typed in the search field and click to "Suche" 
- All Search result appear

![Image](https://user-images.githubusercontent.com/117288240/212850269-c447ec8e-fb5f-46b9-9bfe-2cfa4a2cdf9f.png)

**Comment by balsih** [comment]
@Dummylearners this seems to be a dublicate of [](https://github.com/AkrosAG/Akros-Marketplace/issues/286)

**Comment by LanaKast** [comment]
Yes, it is. the ticket will be combined with #286 in the upcoming backlog refinement
